### PR TITLE
Regenerate code based on new Satori API - Delete Identity

### PR DESCRIPTION
**Pull request by flaviofernandes004**
depends on: https://github.com/heroiclabs/satori/pull/334

### CODE REVIEW

**Comment by lugehorsam** [comment]
We'll also need to add the code the the `client.ts` file and the client interface that invokes this generated code.

**Comment by flaviofernandes004** [comment]
Thx @lugehorsam, I will do those changes as well.

File: packages/satori-js/client.ts [hunk 1]
```diff
@@ -270,4 +270,16 @@ export class Client {
       return Promise.resolve(response !== undefined);
     });
   }
+
+    /** Delete an Identity. */
+    async deleteIdentify(session: Session, id: string) {
```
Comment (lugehorsam) [inline packages/satori-js/client.ts]: should the function name be `deleteIdentify` or `deleteIdentity`?

Comment (flaviofernandes004) [reply packages/satori-js/client.ts]: I think you are looking at an outdated version of the PR

Comment (flaviofernandes004) [reply packages/satori-js/client.ts]: the commit that handles that incoherence is already here afc3c92974f11420e815f32a09a730a361afbf0c

File: packages/satori-js/api.gen.ts [hunk 2]
```diff
@@ -20,6 +20,10 @@ export interface ApiAuthenticateRefreshRequest {
 
 /**  */
 export interface ApiAuthenticateRequest {
+  //Optional custom properties to update with this call. If not set, properties are left as they are on the server.
+  custom?: Record<string, string>;
```
Comment (lugehorsam) [inline packages/satori-js/api.gen.ts:24]: Could you add these as optional parameters to the `authenticate` function? We use the generated diffs as reminders to update the interface.

File: packages/satori-js/client.ts [hunk 3]
```diff
@@ -270,4 +270,16 @@ export class Client {
       return Promise.resolve(response !== undefined);
     });
   }
+
+    /** Delete an Identity. */
+    async deleteIdentity(session: Session, id: string) {
```
Comment (lugehorsam) [inline packages/satori-js/client.ts:275]: The session already has a user ID that you can pass to the inner delete function. I think identity id == user id?

Comment (flaviofernandes004) [reply packages/satori-js/client.ts]: You are right. The backend impl was changed to operate as you say([here](https://github.com/heroiclabs/satori/commit/52b4a1558ebf9f35a05f97f94c20cea78208a384)).

File: packages/satori-js/api.gen.ts (hunk 2 above)
Comment (flaviofernandes004) [reply packages/satori-js/api.gen.ts]: Since this belongs to other endpoint, I will actually remove this change set for now

**Review by lugehorsam — APPROVED** [review]

**Pull request merged**